rust/src/delta.rs

```diff
@@ -1526,6 +1540,36 @@ impl<'a> DeltaTransaction<'a> {
         Ok(PreparedCommit { uri })
     }
 
+    /// Prepare a commit with commit info added
+    pub async fn prepare_commit_with_info(
```

Comment:
Not really :) Main intent was to avoid breaking existing users, but probably in the end nicer to have just one commit method (same for prepare_commit). Will update the PR.

Reply:
Changed it to updating existing commit functions rather than creating new ones.